recipes/libfabric/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.22.0" %}
+{% set build = 0 %}
+
+package:
+  name: libfabric
+  version: {{ version }}
+
+source:
+  url: https://github.com/ofiwg/libfabric/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 302bf1a3ef64bf51a41a1882f87277a7ab259c8790600e30cf78eee25dfa63dd
+
+build:
+  number: {{ build }}
+  skip: true  # [win or osx]
```

Comment:
We'll at least need mac and probably Windows as well, but that can be done on the feedstock

Reply:
We can add these, although I am note sure OFI is relevant for laptops/desktops since libfabric is primarily designed for high-performance computing environments and applications that require fast inter-process communications, often over high-speed networks

Reply:
I agree that single machines don't need the most optimized builds, but OFI is used my mpich and openmpi on macs and by intel mpi on windows already. I think the main goal here is unbundling a dependency already being built/used on conda-forge, so we need to build it everywhere it's being used already if we want to be able to unbundle it from the packages that use it.

Reply:
Could we proceed with merging the current PR for Linux? This will immediately aid users dependent on a functional Linux build. We can look into adding win and osx support on the feedstock afterward.

Reply:
Yes, absolutely. We'll tackle that on the feedstock.

Reply:
Sorry for the misunderstanding, I meant you didn't need to remove these skips until we're on the feedstocks. Put the skip back for `win` at least, since we're not building a Windows package at all here (need a Windows build script).

Reply:
OK